src/tokens/index.ts

```diff
@@ -0,0 +1,42 @@
+import { ChainId } from '../chainIds';
+import { ITokensByChainId } from '../types';
+import * as ethereum from './ethereum';
+import * as rinkeby from './rinkeby';
+import * as bsc from './bsc';
+import * as bscTestnet from './bscTestnet';
+import * as telos from './telos';
+import * as telosTestnet from './telosTestnet';
+import * as polygon from './polygon';
+import * as mumbai from './mumbai';
+import { IToken } from '../interfaces';
+
+export const tokensByChainId: ITokensByChainId = {
+  [ChainId.Ethereum]: ethereum.tokens,
+  [ChainId.Rinkeby]: rinkeby.tokens,
+  [ChainId.BSC]: bsc.tokens,
+  [ChainId.BSCTestnet]: bscTestnet.tokens,
+  [ChainId.Telos]: telos.tokens,
+  [ChainId.TelosTestnet]: telosTestnet.tokens,
+  [ChainId.Polygon]: polygon.tokens,
+  [ChainId.Mumbai]: mumbai.tokens,
+};
+
+export const allTokens: IToken[] = [
```

Comment:
It could also be with .concat() but looking at performance https://javascript.plainenglish.io/efficiently-merging-arrays-in-javascript-32993788a8b2
Looks like spread operator is the fastest for arrays under 10.000 elements